app/src/main/java/com/yeonae/chamelezone/view/place/PlaceReviewTabFragment.kt

```diff
@@ -46,35 +51,52 @@ class PlaceReviewTabFragment : Fragment(), PlaceReviewContract.View {
 
     override fun getMemberCheck(response: Boolean) {
         presenter.getMember()
+        if (response) {
+            review.setOnClickListener {
+                val intent = Intent(context, ReviewCreateActivity::class.java)
+//            intent.putExtra(PLACE_NAME, "$tv_place_name")
+                Log.d("placeNumber", placeNumber.toString())
+                Log.d("PlaceReviewTabFragment placeName", placeName.toString())
+                intent.putExtra(PLACE_NUMBER, placeNumber)
+                intent.putExtra(PLACE_NAME, placeName)
+                startActivity(intent)
+            }
+            btn_more.isVisible = true
+        } else {
+            review.setOnClickListener {
+                val intent = Intent(context, LoginActivity::class.java)
+                startActivity(intent)
+            }
+            btn_more.isInvisible = true
+        }
     }
 
-    private val placeReviewRvAdapter = PlaceReviewTabRvAdapter()
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? =
         inflater.inflate(R.layout.fragment_place_review_tab, container, false)
 
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
         placeNumber = arguments?.getInt(PLACE_NUMBER) ?: 0
-        val placeName = arguments?.getString(PLACE_NAME)
         memberNumber = arguments?.getInt(MEMBER_NUMBER) ?: 0
         Log.d("memberNumber", memberNumber.toString())
 
+        placeReviewRvAdapter = PlaceReviewTabRvAdapter()
+
         setAdapter()
 
         presenter = PlaceReviewPresenter(
-            Injection.reviewRepository(), Injection.memberRepository(requireContext()),this
+            Injection.reviewRepository(), Injection.memberRepository(requireContext()), this
```

Comment:
```suggestion
            Injection.reviewRepository(), Injection.memberRepository(), this
```
application context를 넣어줘야하는데 이렇게 혼용할 우려가 있습니다. 받아서 쓰지말고,
App 의 context를 자체적으로 사용하는게 좋을 것 같아요